source/extensions/tracers/opentelemetry/tracer_stats.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "envoy/stats/stats_macros.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace OpenTelemetry {
+
+#define OPENTELEMETRY_TRACER_STATS(COUNTER)                                                        \
+  COUNTER(http_reports_failed)                                                                     \
+  COUNTER(http_reports_sent)                                                                       \
+  COUNTER(http_reports_success)                                                                    \
+  COUNTER(spans_sent)                                                                              \
+  COUNTER(timer_flushed)
```

Comment:
As part of this PR, you will need to update the docs to reflect the changes to the stats.

Reply:
See my comment below on this new "stats" type https://github.com/envoyproxy/envoy/pull/29207#discussion_r1342529402